projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css

```diff
@@ -1,7 +1,13 @@
 /**
  * The button colors! The colors!
  */
-button {
```

Comment:
wrappers upon wrappers here. I needed a block wrapper in the component to position the dropdown menu to, and I wanted to keep `:host`/`prx-button` itself defaulted to inline, so that's `.wrapper`. I needed to force the primary button and dropdown arrow button into a row, so that's `.primary`.

And just plain `button` was display: relative, but we really only need that for the spinner so I added the `.working` specifier.